pulsar-io/flume/src/test/java/org/apache/pulsar/io/flume/node/TestApplication.java

```diff
@@ -117,7 +117,7 @@ public void testBasicConfiguration() throws Exception {
         eventBus.post(materializedConfiguration);
         application.start();
 
-        Thread.sleep(1000L);
+        Thread.sleep(1003L);
```

Comment:
Is this a meaningful change?

Reply:
no, just to trigger tests. Otherwise, GitHub action won't run.